tests/scripts/nightly/run_gs_only.sh

```diff
@@ -13,4 +13,6 @@ env pytest models/demos/metal_BERT_large_11/tests/test_demo.py
 
 env pytest models/demos/resnet/tests/test_metal_resnet50.py::test_run_resnet50_inference[LoFi-activations_BFLOAT8_B-weights_BFLOAT8_B-batch_20-device_params0]
 
+env pytest models/demos/resnet/tests/test_metal_resnet50.py::test_run_resnet50_2cqs_inference[LoFi-activations_BFLOAT8_B-weights_BFLOAT8_B-batch_20-device_params0]
+
```

Comment:
This file sucks, but can you not do this to set precedence?

Reply:
Separated into separate files